go-controller/pkg/ovn/master.go

```diff
@@ -219,6 +219,7 @@ func (oc *DefaultNetworkController) deleteStaleNodeChassis(node *kapi.Node) erro
 
 // cleanupNodeResources deletes the node resources from the OVN Northbound database
 func (oc *DefaultNetworkController) cleanupNodeResources(nodeName string) error {
+	klog.Infof("Cleaning up node %q OVN network resources", nodeName)
```

Comment:
do you think this log is better placed before return nil on a successful attempt instead at the start?